src/validation/rules/noExplicitString.rule.ts

```diff
@@ -20,24 +19,23 @@ import { stringAliases } from '../../runtypes/specifiedScalarTypes';
  *   field: s
  * }
  *
- *
  */
 export function NoExplicitStringRule(context: ValidationContext): ASTVisitor {
   return {
-    NamedType(node, _1, parent, _2, ancestors) {
-      const castedValue = defaultNamedTypeCast(node);
-      if (stringAliases.includes(castedValue)) {
-        context.reportError(
-          new AnySpecError(
-            `No need to explicitly specify string type since it is the default`,
-            node,
-          ),
-        );
+    FieldDefinition(node, _1, parent, _2, ancestors) {
+      if (node.type.kind === ASTNodeKind.NAMED_TYPE) {
+        if (!node.type.name.value) {
+          return;
+        }
+        if (stringAliases.includes(node.type.name.value)) {
```

Comment:
what about to rewrite into Set?
https://stackoverflow.com/questions/39007637/javascript-set-vs-array-performance/46190569#46190569